backend/src/user_commands.py

```diff
@@ -1,16 +1,25 @@
-# Workload Generator (v1) #
-
 """ Contains dictionary of the commands for each user command that can be specified. 
     each user command is set to this function after decoding the packet in the webserver
-    """
+"""
 """ TODO: 
         - finish all commandws
         - inferface with the SQL database. 
-        - each command needs to log to the specified xml document
+        - logging is out of order -> needs error checking
 """ 
+
+import time
+import queue
+import sys
+sys.path.append('..')
+from audit import logXML as log
 from threading import Lock
 cmd_print_lock = Lock()
 
+buyAmtQ = queue.Queue()
+sellAmtQ = queue.Queue()
+REMOVE = "REMOVE"
+ADD = "ADD"
+
 def printCmd(cmdDict):
```

Comment:
not a big deal, but It looks like all the user commands use both of the following: 

```
cmdDict['timestamp'] = str(int(time.time()))
and 
log.logEvents['accountTransaction' || 'quoteServer' ](cmdDict) || 
```

could we generalize it in this function?

Reply:
oh, maybe not because they have specific error messages?

Reply:
yea it defs need some optimizing, mostly I think bc we havent implemented the commands yet so its a bit unclear which/what actions need to be logged